test/grouped_convnd_bwd_weight/test_grouped_convnd_bwd_weight.cpp

```diff
@@ -27,28 +29,59 @@ class TestGroupedConvndBwdWeight : public ::testing::Test
     using NDimSpatial = std::tuple_element_t<6, Tuple>;
 
     std::vector<ck::utils::conv::ConvParam> conv_params;
-    ck::index_t split_k{2};
+    std::vector<ck::index_t> split_ks{1, 2};
+
+    bool skip_case(const ck::utils::conv::ConvParam& params, const ck::index_t split_k)
+    {
+        // K=1 or C=1 is supported only by dl kernel
```

Comment:
do you mean cases with `K=1 or C=1` or cases `K or C odd`? The comment says something else than the code

Reply:
Fixed to K or C are odd.